src/dispatch_common.c

```diff
@@ -783,6 +783,9 @@ epoxy_egl_get_current_gl_context_api(void)
 {
     EGLint curapi;
 
+    if (!api.egl_handle)
+        return false;
```

Comment:
I don't know anything about this logic, but shouldn't this return EGL_NONE?

Reply:
Yes, now fixed!